Domain/Sia.Shared.Tests/Extensions/HandlerShortCircuitTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Sia.Gateway.Tests.TestDoubles;
+
+namespace Sia.Shared.Tests.Extensions
+{
+    [TestClass]
+    public class HandlerShortCircuitTests
+    {
+        public Task<string> ReturnMockNext()
```

Comment:
It should be possible to declare this in the Handle_Should_Return_Next_If_ShouldRequestContinue_Is_True() method as something like

Task<string> ReturnMockNext() => Task.FromResult("Next result");

Which would make it more clear that it's used only in that method.